### ISSUE TITLE
Blocage des stages de 1 jour

### ISSUE THREAD

**Issue by yoannlecuyer** [EsupPortail/esup-stage#200]
Bonjour,
On m'a fait constaté qu'on ne pouvait pas saisir de stage dont la durée n'était que d'une journée.
En effet, pour certaines de nos formations, les stages peuvent durer un jour voir une demi journée.
Quelle est l'explication derrière cette règle et est-il possible de la modifier ?
Cordialement,
Yoann Lecuyer

**Comment by csylvanie** [comment]
Personne n'en avait fait la demande jusqu'à présent. Je l'enregistre comme évolution. La demi journée sera à gérer au niveau du nombre d'heures.

**Comment by yoannlecuyer** [comment]
Effectivement, la demi journée reste une journée avec 4 heures de travail.
Le blocage est bien sur le fait que la date de fin devrait pouvoir être égale à la date de début du stage.
Merci pour la prise en compte.

**Comment by lwasy44** [comment]
Bonjour,
Nous appuyons la demande de @yoannlecuyer à Nantes Université, à savoir : Le blocage est bien sur le fait que la date de fin devrait pouvoir être égale à la date de début du stage.
Merci 
Laurent

**Comment by lucuhb** [comment]
Bonjour,

nous avons également des stages sur une journée, il faudrait donc pouvoir avoir la même date de début et de fin de stage.

Merci
Lucie

**Comment by csylvanie** [comment]
Demande enregistrée

**Comment by csylvaniepro-tech** [comment]
Livrée en 3.2.0, en 3.2.3 et 3.2.4
